app/src/main/java/ch/epfl/skysync/components/FlightDetail.kt

```diff
@@ -209,6 +263,25 @@ fun ConfirmFlightLocation(confirmedFlight: ConfirmedFlight, padding: Dp, cardCol
   }
 }
 
+/**
+ * Composable function for displaying the takeoff and landing location of a finished flight.
+ *
+ * @param finishedFlight The finished flight object containing details to be displayed.
+ * @param padding The padding value to be applied around the content.
+ * @param cardColor The color to be used for the card background.
+ */
+@Composable
+fun FinishedFlightLocation(finishedFlight: FinishedFlight, padding: Dp, cardColor: Color) {
```

Comment:
This should be 2 separated items. You can use the following function, create 2 items with the right arguments: `"Takeoff location"`, `light.takeOffLocation.name` and `"Landing location"`, `flight.landingLocation.name`
```kotlin
@Composable
fun FinishedFlightLocation(title: String,locationName: String, padding: Dp, cardColor: Color) {
    Card(colors = CardDefaults.cardColors(cardColor), modifier = Modifier.fillMaxWidth()) {
        LargeTitle(title = title, padding = padding, color = Color.Black)
        HyperLinkText(location = locationName, padding = padding)
    }
}
```

Reply:
Done